### PR TITLE
Readme with Instruction content.

### PR DESCRIPTION
**Pull request by theGaryLarson**
**Can use some minor editing.**

- [x] Sharper images on some of the instructions.
- [x]  Perhaps a small blurb up top before instructions.

> Please look it over and give me any critiques that will make it easier to follow.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,79 @@
...
+
+7. From your GitHub Profile submit a pull request
+
+   ![](readme_images/pullRequest.png)
+
+---
+### Terminal Commands
+---
+**Switch to the Git Bash terminal**
+
```
Comment (KristinJJ) [inline README.md:49]: Could the Terminal Commands section also be indented to match the GitHub UI section above, and to make it clearer/look nicer?

File: README.md [hunk 2]
```diff
@@ -1,19 +1,79 @@
...
+> Continue with **step 7** above
+---
 
-Happy to have yall here <3
+Happy to have yall here :heart:
 
-Good luck :)
+Good luck :smile:
 
 You got this
```
Comment (KristinJJ) [inline README.md:92]: We might want to eventually remove Raahil's name because people won't know who Raahil is?

Comment (KristinJJ) [inline README.md:92]: Maybe there could be an additional section at the bottom for reference links, like for Markdown? This is a good one for commonly used Markdown syntax for READMEs https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax

**Review by KristinJJ — APPROVED** [review]

**Comment by KristinJJ** [comment]
I like the suggestions you made about replacing some of the lower resolution images and adding a blurb at the top!

File: README.md (hunk 2 above)
Comment (KristinJJ) [reply README.md:92]: Some other potential links to go into a reference section:

Link to Beck’s Git Intro Video: [Working Together with Github - 30 Minute Virtual TechBytes](https://www.youtube.com/watch?v=UDlIJkvOOPw&list=PL6k4gMRDdFjhHvq9Oulmyph2LezQMU4lg&index=2&t=2s)
Link to Fork Me Repo: https://github.com/atapas/fork-me
Link to GitHub Profile Generator: https://rahuldkjain.github.io/gh-profile-readme-generator/
Link to Good First Issue: https://goodfirstissues.com/

Comment (KristinJJ) [reply README.md:92]: Or could sign it as "Google Developer Student Club at North Seattle College"

File: README.md (hunk 1 above)
Comment (theGaryLarson) [reply README.md:49]: done

File: README.md (hunk 2 above)
Comment (theGaryLarson) [reply README.md:92]: and done :smile:

Comment (theGaryLarson) [reply README.md:92]: and also done :smile: Replaced step 5 reference with GitHub markdown reference listed above.
Also included Beck's video up top. 🎉

Comment (KristinJJ) [reply README.md:92]: @theGaryLarson I also had an idea that if we're talking about collaboration, Open Source, and the Good First Issue tag, we should have an example of the Good First Issue tag. I created a sample issue with the Good First Issue tag (you can find in the Issues tab).

**Comment by KristinJJ** [comment]
@theGaryLarson looks like you have a merge conflict, please fix and then I will merge!

File: README.md [hunk 3]
```diff
@@ -1,20 +1,93 @@
...
+---
+<a href="https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax">GitHub Markdown Reference Guide</a><br>
+Link to Beck’s Git Intro Video: <a href="https://www.youtube.com/watch?v=UDlIJkvOOPw&list=PL6k4gMRDdFjhHvq9Oulmyph2LezQMU4lg&index=3&t=2s">Working Together with Github - 30 Minute Virtual TechBytes</a><br>
+Link to GitHub Profile Generator: https://rahuldkjain.github.io/gh-profile-readme-generator/<br>
+Link to Good First Issue: https://goodfirstissues.com/<br>
+Link to Fork Me Repo: https://github.com/atapas/fork-me with <a href="https://www.youtube.com/watch?v=h8suY-Osn8Q">YouTube video walkthrough</a><br>
+=======
 Happy to have yall here :heart:
 
 Good luck :smile:
```
Comment (ReneCapella) [inline README.md:90]: I don't agree with this phrase. I like "you got this"-- good luck feels like.. *this is so hard, good luck!* you know? Luck has nothing to do with the scenario.

File: README.md [hunk 4]
```diff
@@ -1,20 +1,93 @@
...
+---
+### References
+---
+<a href="https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax">GitHub Markdown Reference Guide</a><br>
+Link to Beck’s Git Intro Video: <a href="https://www.youtube.com/watch?v=UDlIJkvOOPw&list=PL6k4gMRDdFjhHvq9Oulmyph2LezQMU4lg&index=3&t=2s">Working Together with Github - 30 Minute Virtual TechBytes</a><br>
+Link to GitHub Profile Generator: https://rahuldkjain.github.io/gh-profile-readme-generator/<br>
+Link to Good First Issue: https://goodfirstissues.com/<br>
+Link to Fork Me Repo: https://github.com/atapas/fork-me with <a href="https://www.youtube.com/watch?v=h8suY-Osn8Q">YouTube video walkthrough</a><br>
+=======
 Happy to have yall here :heart:
```
Comment (ReneCapella) [inline README.md:88]: "yall" is "y'all"

File: README.md [hunk 5]
```diff
@@ -1,20 +1,93 @@
+Hello everyone and welcome to our club repository! This is a generalized Git Workflow to help gain the skills to collaborate on open source projects. Here you can follow the instructions below to learn how to make a copy of a project from a public repository. Then create changes in your copy on GitHub or download it to your computer to make changes. Last, submit those changes back to the original repository for approval.
```
Comment (ReneCapella) [inline README.md:1]: These are big blocks of text that I don't think anyone will read: can we break these up?

<bolded>Hello...welcome...repository!
<new paragraph, space> This is a generalized....projects.
<white space>
<ordered list>
1.) Follow the instructions below
2.)Create your changes either in the Github UI or locally
3.)Submit changes in a PR to the original repo

File: README.md [hunk 6]
```diff
@@ -1,20 +1,93 @@
+Hello everyone and welcome to our club repository! This is a generalized Git Workflow to help gain the skills to collaborate on open source projects. Here you can follow the instructions below to learn how to make a copy of a project from a public repository. Then create changes in your copy on GitHub or download it to your computer to make changes. Last, submit those changes back to the original repository for approval. 
 
+The **GitHub UI Instructions** will allow you to follow along and make changes within the browser. However you are encouraged to take a deeper dive and use the Git Bash **Terminal commands**. This will allow you to work on the project within your preferred IDE on your computer.  If this is something you want to try just follow along with the additional steps listed within step 2. Good luck!
```
Comment (ReneCapella) [inline README.md:3]: same: smaller sentences, utilize white space, use lists and bolding.

Folks will be in the event while they're here: they won't' be reading bc they'll be listening

File: README.md [hunk 7]
```diff
@@ -1,20 +1,93 @@
...
 
+If you have any questions or need help you can message us within our <a href="https://discord.gg/xfeXTySSbX" target="_blank">Club Discord</a>.
```
Comment (ReneCapella) [inline README.md:5]: Short: 
Questions or thoughts? Connect with us on Discord!...

**Review by ReneCapella — COMMENTED** [review]
This will be a helpful asset.
In the future, definitely delegate to a content person on the project.

The main take away from my feedback is to condense words (be direct) and add white space.

White space allows eyes to navigate a document.
I gave some examples in my review comments.

🔥 love having documentation, Gary. Great idea

**Comment by ReneCapella** [comment]
@KristinJJ Please don't merge until all reviews are finished. Thanks!

**Pull request merged**